dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildProjectDirectory)\..\..\FeatureAreaFilterForInnerLoop.props" />
```

Comment:
What controls this .props file only getting loaded in the InnerLoop.sln? Add a comment, because it is not obvious that this does not always get imported.

Reply:
updated it to load always.